main.go

```diff
@@ -59,6 +59,8 @@ type Config struct {
 	Draft         string          `env:"draft,opt[yes,no]"`
 	PreRelease    string          `env:"pre_release,opt[yes,no]"`
 	FilesToUpload string          `env:"files_to_upload"`
+	APIUrl        string          `env:"api_base_url"`
```

Comment:
Add the `required` attribution to the APIURL and to the UploadURL inputs

Reply:
Fixed.